src/providers/kafka-service.ts

```diff
@@ -0,0 +1,97 @@
+import { Component } from '@angular/core'
+import { NavController } from 'ionic-angular'
+import { Injectable } from '@angular/core'
+import { Observable } from 'rxjs/Observable'
+import 'rxjs/add/operator/map'
+import { Http, Response } from '@angular/http'
+
+import * as KafkaRest from 'kafka-rest/lib/client'
+var Kafka = new KafkaRest({ 'url': 'https://radararmt.ddns.net/kafka' }) //change to production or development url 
+
+
+@Injectable()
+export class KafkaService {
+
+ private aRMTIdSchema:string;
+ private aRMTInfoSchema:string;
+ 
+ private url = 'assets/data/schema/schemas.json'
+ private phq8schema : any;
+
+  constructor (
+    private http: Http
+  ) {
+  }
+
+
+
+ submitQuestionare(values):Promise<any>
+ {
+    return new Promise((resolve,reject) => {
+
+    this.getSchema().subscribe(
+    function(schema) { this.phq8schema = schema },
+    function(error) { console.log("Error at" + error)},
+    function() {
+
+      this.aRMTIdSchema = new KafkaRest.AvroSchema({"type": 'int'})
+      this.aRMTInfoSchema = new KafkaRest.AvroSchema(this.phq8schema)
+
+      Kafka.topic("active_questionnaire_phq8").produce(this.aRMTIdSchema,this.aRMTInfoSchema,
+      {'key':1,'value': {
```

Comment:
@fnobilia just for clarification -> does every questionnaire have a separate topic? (@abhiglobalistic) if not, I would rename the topic to just `active_questionnaire`, if yes, we need to add this to the topic names to the firebase config file (Please add this as feature to the project).